### ISSUE TITLE
error :  Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by curlneuron** [cmusphinx/g2p-seq2seq#172]
when I use this tool to train a model with own train.dict, here comes out the error of core dump,
but it did't dump in earlier steps. Anyone knows the reason?
---------------------------
and here is the code.
g2p-seq2seq --train dict_dir/mad.train --model_dir mymdl_dir/mandarin/
----------------------------------------------------------------------------------
INFO:tensorflow:loss = 0.014552901, step = 5501 (76.460 sec)
INFO:tensorflow:global_step/sec: 1.29732
INFO:tensorflow:loss = 0.03983097, step = 5601 (77.084 sec)
INFO:tensorflow:global_step/sec: 1.29314
INFO:tensorflow:loss = 0.017567342, step = 5701 (77.329 sec)
Segmentation fault (core dumped)

-----------------------------------------------------------------------------------

**Comment by ljhou** [comment]
tensorflow version is 1.11.0 , tensor2tensor version is 1.6.6
